Produkcja

elabrodecka opened the pull request:
Dodanie formularzy ewidencji maszyn i narzedzi, poprawki grafiki i dodanie danych

elabrodecka committed:
zmiany

elabrodecka committed:
Merge branch 'produkcja' of https://github.com/DamianSzczesniak/Projekt into ela

elabrodecka committed:
zmiany2

KamilRajkowski committed:
Poprawione obiążenia string->date bo nie każdy system tak samo wczytuje stringi

KamilRajkowski committed:
Merge branch 'produkcja' of https://github.com/DamianSzczesniak/Projekt into produkcja

Commit:
Poprawki z zajęć

emzi93 committed:
Robocza wersja formularza zamowienie materialu

mateuszkalisz committed:
dodanie formularza ewidencja narzedzia

elabrodecka committed:
zmiany rozmiarów w grafice

elabrodecka committed:
widok 8 dla produkcji - PRCOWNICY_W_PRACY

PiotrStrozekWAT committed:
Poprawione CRUD PRACOWNICY

mateuszkalisz committed:
dodanie narzedzia - dodanie nowego/czyszczenie formularza

elabrodecka committed:
dodanie i poprawki danych

PiotrStrozekWAT committed:
Przeniesiono formularz WykazPracowników do Formularz Kadry

PiotrStrozekWAT committed:
Usunięcie niepotrzebnych plików

mateuszkalisz committed:
przeniesienie narzedzi do ewidencji, dodanie przyciskow do maszyn w ewidencji (podwojne)

PiotrStrozekWAT committed:
Poprawiono zapis edytowanego rekordu

abcek committed:
ewidencja narzedzi i maszyn

MirekLis committed:
do formularza kadry dołozono panel zarządzania urlopami oraz połączono combobox typ urlopu ze słownikiem

PiotrStrozekWAT committed:
Dodano opcję dodawania Wolnego pracownikowi. Usunięto powielone formularze.

PiotrStrozekWAT committed:
Przygotowano formularze pod podstawowe funkcje działu Kadry

kpawlowska1 committed:
zmiany w kreator faktur

PiotrStrozekWAT committed:
Usprawnienia formularza pracownicy

MarcinMaraszek committed:
Zmiana rodzielczości formularzy

rudnickirobert commented on PROJEKTapp/PROJEKTapp/Forms_Produkcja/FormEwidencja.cs:
```diff
@@ -12,14 +12,166 @@ namespace PROJEKTapp.Forms_Produkcja
+            narzedzia.KOSZT_ZA_H = decimal.Parse(this.textBox4.Text);
+
+
+            foreach (Control control in this.panelNarzedzia.Controls)
```
pętla która nic nie robi, do usuniecia

rudnickirobert commented on PROJEKTapp/PROJEKTapp/Forms_Produkcja/FormEwidencja.cs:
```diff
@@ -12,14 +12,166 @@ namespace PROJEKTapp.Forms_Produkcja
+            this.Aktualizuj.Text = "Edytuj";
+            this.WyczyscDane.Text = "Wyczyść dane";
+
+            this.ZapiszM.Text = "Zapisz";
```
zmienic nazwe przycisku

rudnickirobert replied:
nazwa przycisku nadal nie jest zmieniona...

rudnickirobert commented on PROJEKTapp/PROJEKTapp/Forms_Produkcja/FormEwidencja.cs:
```diff
@@ -12,14 +12,166 @@ namespace PROJEKTapp.Forms_Produkcja
+            textBox10.Clear();
+        }
+
+        private void panelMaszyny_Paint(object sender, PaintEventArgs e)
```
puste funkcje które nic nie robią, do usunięcia

rudnickirobert commented on PROJEKTapp/PROJEKTapp/Forms_Produkcja/FormEwidencja.cs:
```diff
@@ -12,14 +12,166 @@ namespace PROJEKTapp.Forms_Produkcja
+            
+
+ 
+            foreach (Control control in this.panelMaszyny.Controls)
```
jw

rudnickirobert commented on PROJEKTapp/PROJEKTapp/Forms_Produkcja/FormGabaryty.cs:
```diff
@@ -61,5 +61,10 @@ private void txtNazwaProduktu_TextChanged(object sender, EventArgs e)
             }
         }
+
+        private void lblTryb_Click(object sender, EventArgs e)
```
pusta funkcja, do usuniecia

MirekLis committed:
dodane nowe stawki i przypisanie stawek do pracowników

rudnickirobert commented on PROJEKTapp/PROJEKTapp/Forms_Produkcja/FormObciazenie.cs:
```diff
@@ -38,26 +39,31 @@ private void ComboBox1_SelectedIndexChanged(object sender, EventArgs e)
             }
         }
+
+        private void GridObciazenie_CellContentClick(object sender, DataGridViewCellEventArgs e)
```
jw

PiotrStrozekWAT committed:
Poprawki do komentarzy

MirekLis committed:
dodano stawki pracownikom pracującym charytatywnie

PiotrStrozekWAT committed:
Merge branch 'Kadry' of https://github.com/KWZP/Projekt into Kadry

PiotrStrozekWAT committed:
Merge branch 'master' into Kadry
	PROJEKTapp/PROJEKTapp/obj/Debug/DesignTimeResolveAssemblyReferencesInput.cache

PiotrStrozekWAT committed:
Merge pull request #13 from KWZP/Kadry

Aktualizacja formularza Kadry i Finanse

kpawlowska1 committed:
zmiany w kretorze faktur

elabrodecka committed:
ewidencja

PiotrStrozekWAT committed:
Poprawki z zajęć

PiotrStrozekWAT committed:
Merge branch 'master' of https://github.com/KWZP/Projekt into Kadry

PiotrStrozekWAT committed:
Usprawnienia i poprawki

DamianSzczesniak committed:
Zmiana Formularza ewidencji Firm

PiotrStrozekWAT committed:
Dodano możliwość zmiany stanowiska pracownika na nowe - stare stanowisko przestaje obowiązywać w dniu rozpoczęcia nowego stanowiska
Zmieniono wybieranie danych przez Select w c# na odwołanie do obiektu PRACOWNICY_ZATRUDNIENI

elabrodecka committed:
formularze spis maszyn i narzedzi

DamianSzczesniak committed:
Odniesienie do jednej instancji bd.

PiotrStrozekWAT committed:
Usunięto zbędne zakomentowane linijki, Zmieniono odwołanie do combobox przy pobieraniou wartości.

rudnickirobert committed:
Merge pull request #18 from KWZP/Kadry

Kadry

DamianSzczesniak committed:
Merge branch 'master' into Kierownik

# Conflicts:
#	PROJEKTapp/PROJEKTapp/PROJEKTapp.csproj
#	PROJEKTapp/PROJEKTapp/obj/Debug/DesignTimeResolveAssemblyReferencesInput.cache

PiotrStrozekWAT committed:
Poprawiono widok PRACOWNICY_ZATRUDNIENI, aby nie ukrywał przyszłych pracowników

PiotrStrozekWAT committed:
Przygotowano możłiwość edycji istniejącego stanowiska lub zmiany na nowe. Bład zapisu edytowanego praocwnika ze względu na ID_STANOWISKA? "nie może być modifikowane" mimo że nie jest zmieniane

rudnickirobert committed:
Merge pull request #16 from KWZP/Kierownik

Zmiana Formularza ewidencji Firm

PiotrStrozekWAT committed:
Dodano wyświetlanie wszystkich adresów pracownika.
Dodano przycisk do wyświetlania szczegółów pracownika (bez akcji)

abcek committed:
ewidencja cd

kpawlowska1 committed:
usuniecie pustych funkcji

kpawlowska1 committed:
Revert "usuniecie pustych funkcji"

This reverts commit 25f296d18e85a95cc9e9b189ee9cf00725fd80ba.

kpawlowska1 committed:
dodanie warunku do kreatora faktur, usuniecie pustych funkcji

MarcinMaraszek committed:
zmiany formularz kreator faktur

MarcinMaraszek committed:
Dodanie widoku PRZYGOTOWANIE_TRANSPORTU

DamianSzczesniak committed:
Poprawa modelu bazy. Usuniecie wywyoływania widoków przy tworzeniu. Dodano wstępny mechanizm logowania. Dodano plik .txt z przykladowymi kontami do repozytorium.

DamianSzczesniak committed:
Haslo jako *************

rudnickirobert committed:
Merge pull request #19 from KWZP/Kierownik

Poprawa modelu bazy. Usuniecie wywoływania widoków przy tworzeniu bazy. D…

rudnickirobert commented on PROJEKTapp/PROJEKTapp/Forms_Produkcja/FormEwidencja.cs:
```diff
@@ -12,14 +12,204 @@ namespace PROJEKTapp.Forms_Produkcja
+
+        private void WyczyscDaneM_Click(object sender, EventArgs e)
+        {
+            textBox5.Clear();
```
proszę nazwać textBoxy adekwatnie do ich funkcji

rudnickirobert commented on PROJEKTapp/PROJEKTapp/Forms_Produkcja/FormEwidencja.cs:
```diff
@@ -12,14 +12,204 @@ namespace PROJEKTapp.Forms_Produkcja
+
+        }
+
+        private void panelNarzedzia_Paint(object sender, PaintEventArgs e)
```
puste funkcje do usunięcia

rudnickirobert commented on PROJEKTapp/PROJEKTapp/Forms_Produkcja/FormEwidencja.cs:
```diff
@@ -12,14 +12,204 @@ namespace PROJEKTapp.Forms_Produkcja
+
+        }
+
+        private void textBox10_TextChanged(object sender, EventArgs e)
```
jw

rudnickirobert commented on PROJEKTapp/PROJEKTapp/Forms_Produkcja/FormWykorzystaneMaszyny.cs:
```diff
@@ -69,5 +70,15 @@ private void lblNazwaProduktu_Click(object sender, EventArgs e)
 
         }
+
+        private void GridWykorzystaneMaszyny_CellContentClick(object sender, DataGridViewCellEventArgs e)
```
znowu puste funkcje

rudnickirobert commented on PROJEKTapp/PROJEKTapp/Forms_Produkcja/FormZmiana_Maszyny.cs:
```diff
@@ -0,0 +1,83 @@
+            db.SaveChanges();
+        }
+
+        private void label6_Click(object sender, EventArgs e)
```
puste funkcje + nazewnictwo komponentów

DamianSzczesniak committed:
Zmiany w bazie,  praca nad aktualizacją statusów zleceń

DamianSzczesniak committed:
Aktualizacja danych w bazie, few bug fixed

DamianSzczesniak committed:
Rozwój podglądu odpowiednich statusów zleceń przez dany dział firmy.
What's new:
- Fixed bugs
- Added more bugs to fix later.

DamianSzczesniak committed:
Merge branch 'master' into kierownik

DamianSzczesniak committed:
Podgląd aktualnego statusu zlecen, przygotowanie do podgladu adekwatnego do uprawnień

DamianSzczesniak committed:
Widoczność statusów w zależności od uprawnień.

DamianSzczesniak committed:
AKTULNY STATUS + ZMIANA STATUSU
Added more BUGS to fix later

DamianSzczesniak committed:
Zmiana Statusu zlecenia tylko w ramach uprawnień.

rudnickirobert committed:
Merge pull request #20 from KWZP/Kierownik

Kierownik

DamianSzczesniak committed:
PRzycisk wylogownaia się dla ułatwienia pracy

DamianSzczesniak committed:
wspomaganie prdukcji part 1

DamianSzczesniak committed:
Merge branch 'master' into kierownik

PiotrStrozekWAT committed:
Dodano możłiwość edycji wskazanego adresu pracownika oraz dodawanie kolejnego adresu.
Poprawka w widoku ADRESY_PRACOWNIKA

PiotrStrozekWAT committed:
Dodano możłiwość dodawania szkolenia oraz przypisania szkolenia do pracownika (wersja wstępna)

PiotrStrozekWAT committed:
Merge branch 'master' into Kadry

PiotrStrozekWAT committed:
Poprawki po operacji merge z master

Problem z zalogowaniem do aplikacji

PiotrStrozekWAT committed:
Usunięto select po tworzeniu widoku

PiotrStrozekWAT committed:
Poprawki błędów po merge z master

PiotrStrozekWAT committed:
Zmieniono kolejność tabulacji w panleu logowania do aplikacji

DamianSzczesniak committed:
Merge remote-tracking branch 'origin/Kadry' into kierownik

DamianSzczesniak committed:
resources po margu

DamianSzczesniak committed:
Conflicts resolved by hard marge

DamianSzczesniak committed:
Merge remote-tracking branch 'origin/LogistykaForm' into hard_marge

# Conflicts:
#	PROJEKTapp/PROJEKTapp/FAKTURY.cs
#	PROJEKTapp/PROJEKTapp/FormLogowanie.resx
#	PROJEKTapp/PROJEKTapp/Forms/FormLogistyka.Designer.cs
#	PROJEKTapp/PROJEKTapp/Forms/FormLogistyka.cs
#	PROJEKTapp/PROJEKTapp/Forms/FormLogistyka.resx
#	PROJEKTapp/PROJEKTapp/Forms/FormZmianaStanuMagazynu.Designer.cs
#	PROJEKTapp/PROJEKTapp/Forms/FormZmianaStanuMagazynu.resx
#	PROJEKTapp/PROJEKTapp/Forms/Forms_Logistyka/FormListaFaktur.resx
#	PROJEKTapp/PROJEKTapp/Forms/Forms_Logistyka/FormLogistyka.Designer.cs
#	PROJEKTapp/PROJEKTapp/Forms/Forms_Logistyka/FormLogistyka.cs
#	PROJEKTapp/PROJEKTapp/Forms/Forms_Logistyka/FormLogistyka.resx
#	PROJEKTapp/PROJEKTapp/Forms/Forms_Logistyka/FormZmianaStanuMagazynu.Designer.cs
#	PROJEKTapp/PROJEKTapp/Forms/Forms_Logistyka/FormZmianaStanuMagazynu.cs
#	PROJEKTapp/PROJEKTapp/Forms/Forms_Logistyka/FormZmianaStanuMagazynu.resx
#	PROJEKTapp/PROJEKTapp/Forms/Forms_Logistyka/KreatorFaktur.Designer.cs
#	PROJEKTapp/PROJEKTapp/Forms/Forms_Logistyka/KreatorFaktur.resx
#	PROJEKTapp/PROJEKTapp/Forms/Forms_Logistyka/PrzygotowanieTransportu.Designer.cs
#	PROJEKTapp/PROJEKTapp/Forms/Forms_Logistyka/PrzygotowanieTransportu.cs
#	PROJEKTapp/PROJEKTapp/Forms/Forms_Logistyka/PrzygotowanieTransportu.resx
#	PROJEKTapp/PROJEKTapp/Forms/Forms_Logistyka/ZamowienieMaterialu.Designer.cs
#	PROJEKTapp/PROJEKTapp/Forms/KreatorFaktur.Designer.cs
#	PROJEKTapp/PROJEKTapp/Forms/KreatorFaktur.resx
#	PROJEKTapp/PROJEKTapp/Forms/PrzygotowanieTransportu.Designer.cs
#	PROJEKTapp/PROJEKTapp/Forms/PrzygotowanieTransportu.cs
#	PROJEKTapp/PROJEKTapp/Forms/PrzygotowanieTransportu.resx
#	PROJEKTapp/PROJEKTapp/Forms/ZamowienieMaterialu.Designer.cs
#	PROJEKTapp/PROJEKTapp/Forms/ZamowienieMaterialu.cs
#	PROJEKTapp/PROJEKTapp/Forms/ZamowienieMaterialu.resx
#	PROJEKTapp/PROJEKTapp/Froms_Logistyka/FormListaFaktur.resx
#	PROJEKTapp/PROJEKTapp/Froms_Logistyka/FormLogistyka.Designer.cs
#	PROJEKTapp/PROJEKTapp/Froms_Logistyka/FormLogistyka.cs
#	PROJEKTapp/PROJEKTapp/Froms_Logistyka/FormLogistyka.resx
#	PROJEKTapp/PROJEKTapp/Froms_Logistyka/FormZmianaStanuMagazynu.Designer.cs
#	PROJEKTapp/PROJEKTapp/Froms_Logistyka/FormZmianaStanuMagazynu.cs
#	PROJEKTapp/PROJEKTapp/Froms_Logistyka/FormZmianaStanuMagazynu.resx
#	PROJEKTapp/PROJEKTapp/Froms_Logistyka/KreatorFaktur.Designer.cs
#	PROJEKTapp/PROJEKTapp/Froms_Logistyka/KreatorFaktur.resx
#	PROJEKTapp/PROJEKTapp/Froms_Logistyka/PrzygotowanieTransportu.Designer.cs
#	PROJEKTapp/PROJEKTapp/Froms_Logistyka/PrzygotowanieTransportu.cs
#	PROJEKTapp/PROJEKTapp/Froms_Logistyka/ZamowienieMaterialu.Designer.cs
#	PROJEKTapp/PROJEKTapp/PROJEKTapp.csproj
#	PROJEKTapp/PROJEKTapp/obj/Debug/DesignTimeResolveAssemblyReferencesInput.cache
#	PROJEKTapp/PROJEKTapp/obj/Debug/PROJEKTapp.Form1.resources
#	PROJEKTapp/PROJEKTapp/obj/Debug/PROJEKTapp.Properties.Resources.resources

DamianSzczesniak committed:
Wtępne zmiany w bazie danych

rudnickirobert committed:
Merge pull request #22 from KWZP/Kadry

Kadry

rudnickirobert committed:
Merge pull request #15 from KWZP/LogistykaForm

Logistyka form

DamianSzczesniak committed:
Kolejne zmiany w bazie danych, usunięcie kilku bugów, prace nad obsługą magazynu produktów, obsługą magazynu surowców, zmianą hasła użytkownika, ewidencją użytkowników,

DamianSzczesniak committed:
powoli do przodu z taskami

DamianSzczesniak committed:
end of the day commit

DamianSzczesniak committed:
What's new : ewidencja kont uzytkowników z poziomu mastera + możliwośc zmiany hasła z poziomu konta

DamianSzczesniak committed:
Wyświtlanie stanu Magazynu Materiałów i stanu Magazynu Surówców, bez możliwości dodania akcji i filtrowania danych

DamianSzczesniak committed:
Poprawiono bugi, wstępna filtracja danych magazynu

DamianSzczesniak committed:
Merge branch 'master' into kierownik

DamianSzczesniak committed:
Fwe bug's fixed, jeden formulardz do dodawania produtów i matriałów

DamianSzczesniak committed:
1 Bug fixed, step by step akcje na magazynie

DamianSzczesniak committed:
Poprawki filtracji danych

DamianSzczesniak committed:
przygotowanie do zdejmowanie produktów / materiałów z magazynu

DamianSzczesniak committed:
What's new:
- Zmiana w widokach aby nie wyświetlały stanu magazynu = 0,
- Możliwość zdejmowania produktów i matriałów z magazynu,
- Optymalizacja kodu.

DamianSzczesniak committed:
What's new:
- Few bug's fixed,
- Dodawanie nowych elementów do magazynów, optymalizacja zdejmowania z magazynu.

rudnickirobert committed:
Merge pull request #23 from KWZP/Kierownik

Kierownik

DamianSzczesniak committed:
What's new:
- Przy dodawaniu do magazynu dany produkt może być przypisany tylko do zleceń które go obejmowały.

DamianSzczesniak committed:
POPRAWKI

DamianSzczesniak committed:
What's new:
- optymalizacja kodu,
- pełna funkcjonalonść znacznika zapełnienia lokalizacji,
- few bug's fixed

rudnickirobert committed:
Merge pull request #24 from KWZP/Kierownik

Kierownik

rudnickirobert commented:
@DamianSzczesniak @elabrodecka dlaczego tutaj się nic nie dzieje?

KamilRajkowski commented:
Nic tu się nie dzieje ponieważ większość z nas wyjeżdżała w ciągu ostatniego tygodnia. Niedługo powinny zostać dodane działające formularze z ewidencją maszyn i narzędzi podobnie jak to wygląda z pracownikami oraz poprawki estetyczne jak te new entities czy puste funkcjie. Niestety z mojej strony pojawił się większy problem, który bez skutecznie próbowałem rozwiązać najpierw sam, a później z pomocą innych kierowników polegający na tym, że po z pullowaniu ostatnich zmian (później nawet po klonowaniu repozytorium z dowolnego pkt) visual wyświetla błąd o braku powiązań między tabelami, które w sql serverze istnieją i działają.

mateuszkalisz commented:
Puste funkcje są pousuwane, problem mamy z tym, że w ewidencji maszyn i narzędzi podczas zapisywania wywala błąd:
![image](https://user-images.githubusercontent.com/38287804/39723491-4d537d50-5246-11e8-9785-4e7bb4bbf327.png)

rudnickirobert commented:
@mateuszkalisz wyglada na to jakby db było nullem (nie zainicjalizowane), która to klasa?

MirekLis committed:
ddd

rudnickirobert commented:
@mateuszkalisz nie widzę tego pliku w kodzie więc nie mogę pomóc- proszę o wcommitowanie

mateuszkalisz commented:
Mirka commit naprawił ta zmianę - z pomocą Damiana rozwiązaliśmy wcześniej opisywany problem

DamianSzczesniak committed:
Whats new:
- dodanie widoku do bazy sql, który był, ale zniknął przy mergowaniu.
- exception handler dla edycji danych aktualnie zalogowanego uzytkownika.

DamianSzczesniak committed:
What's new:
- Możliwość podglądu histori zmiany statusu danego zlecenia w raz z datami zmian

DamianSzczesniak committed:
What's new:
- Likwidacja bugów przy kompletowaniu listy produktów do oferty

DamianSzczesniak committed:
Merge branch 'produkcja' of https://github.com/KWZP/Projekt into produkcja

# Conflicts:
#	KWZP_PROJECT_DATABASE_CREATOR.sql
#	PROJEKTapp/PROJEKTapp/Forms/Form1.cs
#	PROJEKTapp/PROJEKTapp/Forms/Form2.Designer.cs
#	PROJEKTapp/PROJEKTapp/PROJEKTapp.csproj
#	PROJEKTapp/PROJEKTapp/obj/Debug/DesignTimeResolveAssemblyReferencesInput.cache

DamianSzczesniak committed:
poprawki na branchu produkcji by kierownik

DamianSzczesniak commented:
Merge może iść automatycznie, Branch aktualny z kierownikiem.

DamianSzczesniak committed:
Usunięcie pustych funkcji i poprawione znalezione błędne nazwy.
